lib/jekyll/scholar/utilities.rb

```diff
@@ -27,6 +27,8 @@ def split_arguments(arguments)
       end
 
       def optparse(arguments)
+        @offset = 0   # set default
```

Comment:
Please don't set default values here. If you need default values set them with a nil guard in the reader method. E.g.:

    def offset
       @offset ||= 0
    end